Composer/packages/server/src/models/connector/csharpBotConnector.ts

```diff
@@ -152,6 +152,8 @@ export class CSharpBotConnector implements IBotConnector {
           `bin/Debug/${envSettings.runtimeFrameworkVersion}/BotProject.dll`,
           `--urls`,
           this.endpoint,
+          `--environment`,
```

Comment:
do we need to update the protocol to add a new parameter

Reply:
> do we need to update the protocol to add a new parameter

dotnet receive the env parameter to determine whether it's under development

Reply:
this param will be sent to 'dotnet run' iteself, not a botproject level param. 
https://stackoverflow.com/questions/50821504/asp-net-core-set-hosting-environment-in-build-process